.github/workflows/testing_coverage.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       # Run pytest with coverage report, saving to xml
       - name: Run tests on python 3.8
         run: |
+          poetry install
```

Comment:
Logically this would more belong to the section above, but does not matter so much.

Reply:
I can't put a run statement with an action. But I will put it as another step.